pkg/util/common.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/manifoldco/promptui"
+)
+
+// Run wraps the exec.Cmd.Run() command and sets the standard output.
+// TODO: Should this take an error code argument/return an error code?
+func Run(c *exec.Cmd) {
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stderr
+	if err := c.Run(); err != nil {
+		log.Fatalf("Command %q failed: %v", strings.Join(c.Args, " "), err)
+	}
+}
+
+/*
+#############################################################################
+# Simple yes/no prompt
+#
+# @optparam default -n(default)/-y/-e (default to n, y or make (e)xplicit)
+# @param message
+common::askyorn () {
+  local yorn
+  local def=n
+  local msg="y/N"
+
+  case $1 in
+  -y) # yes default
+      def="y" msg="Y/n"
+      shift
+      ;;
+  -e) # Explicit
+      def="" msg="y/n"
+      shift
+      ;;
+  -n) shift
+      ;;
+  esac
+
+  while [[ $yorn != [yYnN] ]]; do
+    logecho -n "$*? ($msg): "
+    read yorn
+    : ${yorn:=$def}
+  done
+
+  # Final test to set return code
+  [[ $yorn == [yY] ]]
+}
+*/
+
+func AskYesOrNo(question string) (string, error) {
```

Comment:
I would return a `bool` here to make the usage more straight forward.

Reply:
Done!